float32 heading
float32 range
bool  is_new
